src/test/ui/pattern/move-ref-patterns/move-ref-patterns-dynamic-semantics.rs

```diff
@@ -74,6 +74,6 @@ fn main() {
         };
         lam((mk(19), mk(20), mk(21), mk(22)));
     }
-    let expected = [2, 3, 6, 5, 7, 8, 12, 11, 9, 10, 18, 13, 14, 15, 16, 17, 21, 19, 20, 22, 4, 1];
+    let expected = [2, 3, 6, 5, 7, 8, 12, 11, 9, 10, 13, 18, 14, 15, 16, 17, 19, 21, 20, 22, 4, 1];
```

Comment:
The drop order seems to have changed. I don't know enough about the semantics to know if this is valid.